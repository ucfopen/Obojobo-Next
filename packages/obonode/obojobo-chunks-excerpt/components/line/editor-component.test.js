import React from 'react'
import renderer from 'react-test-renderer'

import Line from './editor-component'

describe('Line Editor Node', () => {
	test('Line builds the expected component', () => {
		const component = renderer.create(<Line element={{ content: {} }} />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
