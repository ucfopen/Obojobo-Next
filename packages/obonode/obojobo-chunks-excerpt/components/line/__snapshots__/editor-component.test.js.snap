// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line Editor Node Line builds the expected component 1`] = `
<span
  className="text align-left"
/>
`;
